src/main/java/marquez/service/models/Datasource.java

```diff
@@ -0,0 +1,20 @@
+package marquez.service.models;
+
+import java.time.Instant;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+import lombok.ToString;
+import marquez.common.models.ConnectionUrl;
+import marquez.common.models.DatasourceName;
+
+@RequiredArgsConstructor
+@EqualsAndHashCode
+@ToString
+public class Datasource {
+
+  @Getter @NonNull private final Instant createdAt;
+  @Getter @NonNull private final DatasourceName dataSourceName;
```

Comment:
minor: it's great that we defined the `DatasourceName` type, but when used in `Datasource`, let's name `dataSourceName` as just `name`: meaning `Datasource.name` reads better than `Datasource.dataSourceName`

Reply:
Sounds good. Will take care of this in the next revision.

Reply:
Please see PRs #394, #387, and #384, where this is fixed.